vscode-dotnet-sdk-extension/package.json

```diff
@@ -0,0 +1,107 @@
+{
+	"name": "vscode-dotnet-sdk",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/dotnet/vscode-dotnet-runtime.git"
+	},
+	"bugs": {
+		"url": "https://github.com/dotnet/vscode-dotnet-runtime/issues"
+	},
+	"license": "MIT",
+	"author": "Microsoft Corporation",
+	"displayName": ".NET SDK Install Tool",
+	"description": "Allows acquisition of the .NET SDK.",
+	"icon": "images/dotnetIcon.png",
+	"version": "0.0.1",
+	"publisher": "ms-dotnettools",
+	"engines": {
+		"vscode": "^1.41.0"
+	},
+	"categories": [
+		"Other"
+	],
+	"keywords": [
+		".NET",
+		".NET Core",
+		"dotnet",
+		".NET SDK"
+	],
+	"activationEvents": [
+		"onCommand:dotnet-sdk.acquire",
+		"onCommand:dotnet-sdk.uninstallAll",
+		"onCommand:dotnet-sdk.showAcquisitionLog",
+		"onCommand:dotnet-sdk.reportIssue"
+	],
+	"main": "./dist/extension.js",
+	"contributes": {
+		"commands": [
+			{
+				"command": "dotnet-sdk.reportIssue",
+				"title": "Report an issue with the .NET SDK Install Tool.",
+				"category": ".NET SDK Install Tool"
+			},
+			{
+				"command": "dotnet-sdk.acquire",
+				"title": "Install the .NET SDK.",
+				"category": ".NET SDK Install Tool"
+			},
+			{
+				"command": "dotnet-sdk.uninstallAll",
+				"title": "Uninstall all VS Code copies of .NET SDK.",
+				"category": ".NET SDK Install Tool"
+			}
+		],
+		"configuration": {
+			"title": ".NET SDK Install Tool",
+			"properties": {
+				"dotnetSDKAcquisitionExtension.enableTelemetry": {
+					"type": "boolean",
+					"default": true,
+					"description": "Enable Telemetry for the .NET SDK install tool."
+				},
+				"dotnetSDKAcquisitionExtension.installTimeoutValue": {
+					"type": "number",
+					"default": 120,
```

Comment:
2mins may not be enough for an SDK installation given they're pretty massive. I don't have a good number in mind but maybe Kathleen would?

Reply:
I've doubled this to 240 for now, @KathleenDollard let me know if you have a better timeout value.